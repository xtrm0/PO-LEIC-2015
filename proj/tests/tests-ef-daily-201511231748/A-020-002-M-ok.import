Objects in Space
Obi-Wan Kenobi/ob1@tatooine.geocities.com
SECTION||A Walk in the Desert
PARAGRAPH|These aren’t the droids you’re looking for.
SECTION||Patience You Must Have
PARAGRAPH|Do or do not.
PARAGRAPH|There is no try.
SECTION||Drama in the Clouds
PARAGRAPH|Obi-Wan never told you what happened to your father.
PARAGRAPH|He told me enough! He told me YOU killed him.
PARAGRAPH|NOOOOOOOOOOOOOOOOOOO...
