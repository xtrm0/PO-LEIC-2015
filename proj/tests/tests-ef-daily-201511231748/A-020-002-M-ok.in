8
4
2
3
11
1
3
0
0
0
