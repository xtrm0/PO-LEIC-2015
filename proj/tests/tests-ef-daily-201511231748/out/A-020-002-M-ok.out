Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Identificador da secção: Nova secção activa: '2'.
Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: [] {Drama in the Clouds}
Obi-Wan never told you what happened to your father.
He told me enough! He told me YOU killed him.
NOOOOOOOOOOOOOOOOOOO...
Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Número do parágrafo: Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: [] {Drama in the Clouds}
Obi-Wan never told you what happened to your father.
NOOOOOOOOOOOOOOOOOOO...
Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: 